proxyd/integration_tests/testdata/multicall.toml

```diff
@@ -0,0 +1,33 @@
+[server]
+rpc_port = 8545
+enable_served_by_header = true
+timeout_seconds = 7
+
+[backend]
+response_timeout_seconds = 4
+max_degraded_latency_threshold = "30ms"
+
+[backends]
+[backends.node1]
+rpc_url = "$NODE1_URL"
+
+[backends.node2]
+rpc_url = "$NODE2_URL"
+
+[backends.node3]
+rpc_url = "$NODE3_URL"
+
+[backend_groups]
+[backend_groups.node]
+backends = ["node1", "node2", "node3"]
+routing_strategy = "multicall"
```

Comment:
are those two configs necessary?
```
routing_strategy = "multicall"
multicall_mode = true
```

what is the difference between them?

Reply:
mutlicall_mode is not used and will be removed. It was used in the first draft implementation